### DISCUSSION TITLE
Variáveis de Ambiente - Homologação e Teste

### DISCUSSION THREAD

**Discussion by ghost** [balta-io/2801#73] [Perguntas e Respostas] [answered]
O ambiente de homologação (staging) é diferente do ambiente de teste?

**Comment by andrebaltieri** [comment, marked as answer]
@PolarisTDM depende do lugar sim, já trabalhei em empresas que tinha um ambiente de testes para os desenvolvedores (Comumente chamado de ambiente Dev), depois stagging e depois produção.
